<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalaryWidget</class>
 <widget class="QWidget" name="SalaryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="mainLayout" stretch="2,10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font-size: 30px;
color: white;</string>
         </property>
         <property name="text">
          <string>Salary Details</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="generateSalaryButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: #274c77;
color:#FFFAFB;
font:bold;
padding: 5px;
border-radius:7px;
width: 100px;
margin-right:15px;
height: 25px;
}

QPushButton:hover {
  background-color: #4cc9f0;
  color:black;
font:bold;
padding: 5px;
border-radius:7px;
}

QPushButton:clicked{
 background-color: rgb(255, 233, 67);
  color:black;
font:bold;
padding: 3px;
border-radius:10px;

}</string>
         </property>
         <property name="text">
          <string>Generate Salary</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="tableLayout" stretch="1,9">
       <item>
        <layout class="QHBoxLayout" name="filterLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="orgField">
           <property name="styleSheet">
            <string notr="true">background-color:#FFFAFB;
border-radius:10px;
margin: 3px 0;
font-size: 16px</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
